catalog/pipeline/push-to-external-registry/0.6/README.md

```diff
@@ -16,7 +16,7 @@ Tekton pipeline to push images to an external registry.
 | addSourceShaTag | When pushing the snapshot components, also push a tag with the image source sha | Yes | true |
 | addTimestampTag | When pushing the snapshot components, also push a tag with the current timestamp | Yes | false |
 | pyxisServerType | The Pyxis server type to use. Options are 'production' and 'stage' | No | - |
-| pyxisSecret | The kubernetes secret to use to authenticate to Pyxis | No | - |
+| pyxisSecret | The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert. | No | - |
```

Comment:
I don't think this should end in a `.` as no other parameters do (same for everywhere it is listed)

Reply:
It felt natural to use full sentences once it's over one sentence long. But you're right, other strings miss the final `.` as well. Fixed.